frontend/apps/reader/modules/readerlink.lua

```diff
@@ -236,11 +236,11 @@ function ReaderLink:registerKeyEvents()
     if Device:hasScreenKB() or Device:hasSymKey() then
         self.key_events.GotoSelectedPageLink = { { "Press" }, event = "GotoSelectedPageLink" }
         if Device:hasKeyboard() then
-            self.key_events.AddCurrentLocationToStack = { { "Shift", "Press" } }
+            self.key_events.AddCustomCurrentLocationToStack = { { "Shift", "Press" }, args = true, }
```

Comment:
Will there be `args = false` anywhere? If not, you don't need it.

Reply:
I do need it, otherwise it won't show the notification. Which is the whole point of the whole thing.

Reply:
If it is always truthy, just remove the condition in the handler.